boards/hamilton/Makefile.features

```diff
@@ -0,0 +1,16 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_cpuid
```

Comment:
This feature is provided by the CPU, you can just add `-include $(RIOTCPU)/samd21/Makefile.features` at the end of this file.
Same for `periph_pm` and `cpp`

Reply:
done